job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/step/MultiJobStepContext.groovy

```diff
@@ -49,6 +49,7 @@ class MultiJobStepContext extends StepContext {
         stepNodes << new NodeBuilder().'com.tikal.jenkins.plugins.multijob.MultiJobBuilder' {
             phaseName phaseContext.phaseName
             delegate.continuationCondition(phaseContext.continuationCondition)
+            delegate.executionType(phaseContext.executionType)
```

Comment:
This needs to be wrapped in an if block testing `jobManagement.isMinimumPluginVersionInstalled(...)` so that the new element only gets generated if the minimum required plugin version is installed.